### ISSUE TITLE
Retry Failed Email causes client error

### ISSUE THREAD

**Issue by spencern** [reactioncommerce/reaction#2112]
Attempting to retry a failed email throws an `i18next is not defined` error.

![image](https://cloud.githubusercontent.com/assets/1203639/24935015/783db72c-1edc-11e7-99a0-3f296092528c.png)

```
Exception in delivering result of invoking 'emails/retryFailed': ReferenceError: i18next is not defined
    at http://localhost:3000/app/app.js?hash=7acddf388b4df18bf1c4f053eee3d059e9adfe96:56555:29
    at MethodInvoker._callback (http://localhost:3000/packages/meteor.js?hash=27829e936d09beae3149ecfbf3332c42ccb1596f:1105:22)
    at MethodInvoker._maybeInvokeCallback (http://localhost:3000/packages/ddp-client.js?hash=89f721bb437611dfd558156033a1367eb42686c0:3557:12)
    at MethodInvoker.dataVisible (http://localhost:3000/packages/ddp-client.js?hash=89f721bb437611dfd558156033a1367eb42686c0:3586:10)
    at http://localhost:3000/packages/ddp-client.js?hash=89f721bb437611dfd558156033a1367eb42686c0:4473:7
    at Array.forEach (native)
    at Function._.each._.forEach (http://localhost:3000/packages/underscore.js?hash=cde485f60699ff9aced3305f70189e39c665183c:149:11)
    at Connection._runAfterUpdateCallbacks (http://localhost:3000/packages/ddp-client.js?hash=89f721bb437611dfd558156033a1367eb42686c0:4472:7)
    at Connection._performWrites (http://localhost:3000/packages/ddp-client.js?hash=89f721bb437611dfd558156033a1367eb42686c0:4462:10)
    at Connection._flushBufferedWrites (http://localhost:3000/packages/ddp-client.js?hash=89f721bb437611dfd558156033a1367eb42686c0:4425:10)
```

**Comment by zenweasel** [comment]
This has got to just be a missing import. I remember fixing a bunch of those.

**Comment by zenweasel** [comment]
Closed via #2132
